resources/views/layouts/admin-lite.blade.php

```diff
@@ -394,7 +394,8 @@ function showPage() {
     immediately after the control sidebar -->
     <div class="control-sidebar-bg"></div>
 </div>
-<div id="loader"></div>
+<div id="loader" class="loader"></div>
+<div id="loader1" class="loader" style="display: none"></div>
```

Comment:
Why two loaders one with display none ?

Reply:
A new loader was added to handle resetting.